src/client/debugger/jupyter/kernelDebugAdapter.ts

```diff
@@ -313,6 +356,19 @@ export class KernelDebugAdapter implements DebugAdapter, IKernelDebugAdapter {
             }
         });
 
+        if ((message as DebugProtocol.StackTraceResponse).command === 'stackTrace') {
```

Comment:
Would be good to have comments around what these methods are doing (or at least at the class level).
When i read this code, I wasn't sure what we were doing & why, i had to build my own notebook debugger (personal extension) to understand out what we were doing & why...